content/platform/data-services-cloud-console-api/home.md

```diff
@@ -0,0 +1,176 @@
+
+Some of the advantages of distributing in OpenAPI 3.0 format:
+
+1. Updates to the API can be generated in a more agile manner where documentation is embedded, describing any endpoints, parameters, and more; such as contact information, license, terms of use. 
+2. Consumers of this API also gain the benefits the agility, using the converter from openAPI yaml, or json to any programming language that were used as part of the their automation or CI/CD workflow. (Please check <https://openapi.tools> for more information for the API tools to generate client code)
+
+## REST API Details
+
+Any one can download this OpenAPI Specification (OAS) v3 definition of the DSCC from the following: [Link to the API repository](https://console-us1.data.cloud.hpe.com/doc/api/v1/)
+
+![DSCC_API_Doc](/img/api-documentation-display.png "DSCC API Documentation")
+
+The website also provides several other information:
+
+1. The list of the REST API resources that are supported as of the release. 
+2. The information about the HTTP method, parameters and the responses that are expected out for this resource
+3. The syntax for the HTTP method and path to this resource. Note that this path is relative path, for the complete path, please add the base-URL documented below to provide the full path.
+4. The body of response is returned in JSON format according to the response status of the REST API.
+
+The website also provides the links to download the DSCC open-API definitions in either json or yaml format. Below is an example of the downloaded json file from the DSCC REST API documentation website.
+
+![](/img/open-api-yaml.png)
+
+### Documentation and Tutorials
+
+The overview of the Data Services Cloud Console is available at [this link](https://www.hpe.com/us/en/storage/data-services-cloud-console.html#overview). There will be more blogs that will provide more information to help customers adopting this API with examples, code snippets and many other helpful information.
+
+### Documented Attributes
+
+* **API Name & Description** - Provides short description of the objective for this API with the supported HTTP request method (POST, GET, DELETE, PATCH, PUT etc).
+* **API Path** - Provides detail URL path as the end-point to issue the API call. Note that user must add the base path URL to extend this path to the correct resource end-point.
+* **API Parameter** - Provides the client to input information such as the object for the manipulation, select a filter to limit the returned objects, and other purposes.
```

Comment:
This bullet should be changed to either:
"Provides the client with the ability to input information, such as the object for manipulation, or to select a filter to limit the returned objects, as well as other purposes."
OR
"**Allows** the client to input information, such as the object f**or m**anipulation, select a filter to limit the returned objects, and other purposes."